tests/data/test_timeseries.py

```diff
@@ -122,3 +123,196 @@ def test_as_cumulative_timeseries_raises_warning(example_timeseries_ingested):
         # Verify warning has been triggered
         assert len(w) == 1
         assert "invalid" in str(w[-1].message)
+
+
+def test_convert_timeseries_to_unit_mwe(example_timeseries_ingested):
+
+    converted_timeseries = example_timeseries_ingested.convert_timeseries_to_unit_mwe()
+    assert converted_timeseries.unit == "mwe"
+    assert example_timeseries_ingested.unit == "m"
+    assert not np.array_equal(converted_timeseries.data.changes, example_timeseries_ingested.data.changes)
+    expected_converted_changes = example_timeseries_ingested.data.changes / 997 * 850
```

Comment:
It would be good to avoid magic numbers like these by creating a module containing well-named constants (for things like the density of water).

I've frequently had times where people have complained about the speed of light (because you get different numbers with different atmospheric models) and I've saved lots of time by being able to reach in and change it in just one place. In addition, Find-and-replace with numbers is significantly less reliable than doing it with long names.

Reply:
For example here, 997 is the density of water at a particular temperature. Is that a sensible temperature for glaciers, ice sheets or ocean? Even if we say "yes", you can bet there's an academic somewhere that will say "well actually..."

Reply:
they are the default values in the function. could make it more obvious and pass them in.

Reply:
I would rather not make them constants as they need to stay flexible

Reply:
happier now? :)

Reply:
So, the key goal is that the number "997", used to represent the density of water, should only occur once in the codebase. It should then be imported wherever it is used as a default setting.

There are two ways of doing this: The "Julia style", which is better for constants you think you'll almost never change, is to create a python module that only contains constants, so something like:

`constants.py` reads:
`DENSITY_OF_WATER_GT_PER_M3 = 997`

Then in other files:

```
import constants

def func(density_of_water = constants.DENSITY_OF_WATER_GT_PER_M3):
    stuff
```

The other way of doing it, let's call it the "EOLIS style", is to set it in a config file which is then read and distributed through the code as a config object of some kind. You can of course do both by instantiating the config object within constants.py, but this is rarely necessary.

But however it's achieved, for me this will be complete once the key goal is achieved: numbers that are constant should only occur once in the codebase, and I should never have to guess what a number I'm looking at represents (i.e. no "magic numbers").

Reply:
Thanks, went for Julia-method, and if we have configs in the end they could overwrite the const values?

Reply:
yes, configs supplied as files would be expected to replace the values in the code, if the individual did not simply want to edit constants.py